.github/workflows/golang-security-scan.yml

```diff
@@ -33,46 +33,41 @@ jobs:
   govulncheck:
     name: Go Vulnerability Check
     runs-on: ubuntu-latest
-    continue-on-error: true
     steps:
       - uses: golang/govulncheck-action@v1
         with:
           go-version-input: ${{ inputs.go-version }}
           go-package: ./...
           work-dir: ${{ inputs.working-directory }}
-    outputs:
-      status: ${{ job.status }}
 
   trivyscan:
     name: Trivy Scan
     runs-on: ubuntu-latest
     if: ${{ inputs.execute-trivy-scan }}
-    continue-on-error: true
     steps:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v4
 
       - name: Build docker image
-        run: docker build --build-arg APP_DEPLOYMENT=Prod -t ga-image-app:latest .
+        run: docker build --build-arg APP_DEPLOYMENT=Prod -t ${{ github.repository }}:latest .
 
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@master
         with:
-          image-ref: "ga-image-app:latest"
+          image-ref: "${{ github.repository }}:latest"
           format: "table"
           exit-code: "1"
           ignore-unfixed: true
           vuln-type: "os,library"
           severity: "MEDIUM,HIGH,CRITICAL"
-    outputs:
-      status: ${{ job.status }}
 
   create_issues:
     name: Create issues for failed jobs
-    needs: [govulncheck, trivyscan]
     runs-on: ubuntu-latest
+    needs: [govulncheck, trivyscan]
+    if: always() && (needs.govulncheck.result == 'failure' || needs.trivyscan.result == 'failure')
```

Comment:
More curious than anything, why do we need `always()` here?

Reply:
Without it the job is skipped if either of the jobs listed for 'needs' are skipped.